src/components/Database/DataViewer/common/fetch.ts

```diff
@@ -139,12 +139,12 @@ function toObservable(query: firebase.database.Query) {
   });
 }
 
-function applyQuery(
+export function applyQuery(
   ref: firebase.database.Reference,
   params: QueryParams
 ): firebase.database.Query {
   const { key, operator, value, limit } = params;
-  if (key && operator && value) {
+  if (key != undefined && operator != undefined && value != undefined) {
```

Comment:
It's unclear here why `!=` is necessary... Perhaps a comment above to clarify that the != is intentional

Reply:
Added the comment. tldr - it prevents bugs when the "value" needs to be false.